grails-app/views/deliveryNote/_printPage.gsp

```diff
@@ -135,13 +124,16 @@
                             </g:if>
                         </td>
                         <td class="center middle">
-                            <g:if test="${picklistItemsGroup}">
+                            <g:if test="${shipmentItem}">
+                                ${shipmentItem?.quantity ?: 0} ${requisitionItem?.product?.unitOfMeasure ?: "EA"}
```

Comment:
Should this column header be `Delivered by container` if there are shipment items? And `Delivered by lot` if the data is pulled from grouped picklist items?